corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -137,6 +137,12 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
         });
 
         $('#select-case').off('click').click(function () {
+            if (hqImport('hqwebapp/js/toggles').toggleEnabled('APP_ANALYTICS')) {
```

Comment:
This will record all case selections, not just those made via case claim. But I chatted with Kat and she said this event can be removed - she wants to track the searches but doesn't need to also track the claims.